modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -53,43 +62,68 @@ public final boolean process(
         RoundEnvironment roundEnv
     ) {
         for (TypeElement annotation : annotations) {
-            List<String> annotatedClasses =
-                getAnnotatedClasses(roundEnv, annotation);
-            makeAnnotatedClassesFile(annotatedClasses);
+            List<List<String>> annotatedElements =
+                getAnnotatedElements(roundEnv, annotation);
+            makeAnnotatedElementsFiles(
+                annotatedElements.get(0), annotatedElements.get(1));
         }
         return true;
     }
 
-    private List<String> getAnnotatedClasses(
+    private List<List<String>> getAnnotatedElements(
         RoundEnvironment roundEnv,
         TypeElement annotation
     ) {
         List<String> annotatedClasses = new ArrayList<String>();
+        List<String> annotatedMethods = new ArrayList<String>();
+        List<List<String>> annotatedElements = new ArrayList<List<String>>();
         for (Element element : roundEnv.getElementsAnnotatedWith(annotation)) {
             List<String> warnings = Arrays.asList(
                     element.getAnnotation(SuppressWarnings.class).value());
             if (warnings.contains("modernizer")) {
                 if (element.getKind().isClass()) {
                     annotatedClasses.add(getClassHeader(element));
+                } else if (element.getKind().equals(ElementKind.METHOD) ||
+                    element.getKind().equals(ElementKind.CONSTRUCTOR)) {
+                    annotatedMethods.add(getMethodIdentifierString(element));
                 }
             }
         }
-        return annotatedClasses;
+        annotatedElements.add(annotatedClasses);
+        annotatedElements.add(annotatedMethods);
+        return annotatedElements;
     }
 
-    private void makeAnnotatedClassesFile(List<String> annotatedClasses) {
-        if (annotatedClasses.isEmpty()) {
+    private void makeAnnotatedElementsFiles(
+        List<String> annotatedClasses,
+        List<String> annotatedMethods
+    ) {
```

Comment:
Rather than having this method take both lists and make both output files, could it take the output file as an arg and then we just call it twice? (once for classes and again for methods)